include/SFML/Network/IpAddress.hpp

```diff
@@ -161,9 +161,11 @@ class SFML_NETWORK_API IpAddress
     ////////////////////////////////////////////////////////////
     // Static member data
     ////////////////////////////////////////////////////////////
+    // NOLINTBEGIN(readability-identifier-naming)
     static const IpAddress Any;       //!< Value representing any address (0.0.0.0)
     static const IpAddress LocalHost; //!< The "localhost" address (for connecting a computer to itself locally)
     static const IpAddress Broadcast; //!< The "broadcast" address (for sending UDP messages to everyone on a local network)
+    // NOLINTEND(readability-identifier-naming)
```

Comment:
Can the rule not be adjusted to not care about public (static) const?

Reply:
Not that I'm aware of. I tried but couldn't carve a rule for these `UpperCase` constants we use. The options are [vast](https://clang.llvm.org/extra/clang-tidy/checks/readability/identifier-naming.html) so hopefully I just missed it.